hacks/install_etcdctl

```diff
@@ -21,28 +21,54 @@ function show_help () {
 }
 
 function install () {
-  local version=$1
+  # renovate: datasource=github-releases depName=etcd-io/etcd
+  _version="v3.5.12"
```

Comment:
name it `etcd_version`, or we should adapt the regex so that it also accepts just `version`

Reply:
I presonally prefer `etcd_version`